scripts/gh_matrix_builder.py

```diff
@@ -51,7 +51,7 @@ def build_debug_config(overrides):
     "coverage": True,
     "llvm_config": "llvm-config-9",
     "clang": "clang-9",
-    "os": "ubuntu-18.04",
+    "os": "ubuntu-20.04",
```

Comment:
Is it possible that `ubuntu-20.04` will not match `ubuntu-latest` in such case?

Reply:
atm it matches ubuntu-latest but once it switches the llvm and clang versions will probably need to be adjusted thats why i pinned all jobs that directly build stuff in ubuntu to a specific version but changed the ones that run the build processes in docker to ubuntu-latest cause they dont rely on any packages from the environment they run in